### PR TITLE
Fix: Design review for pull quotes. Small changes for iOS.

### PR DESCRIPTION
**Pull request by jimhunty**
After a design review with Jake, the following changes have been made.

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/ca9ccf52105b6204

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/5260f30fd24db054

File: packages/pull-quote/styles/index.android.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { StyleSheet } from "react-native";
+import sharedStyles from "./shared";
+
+const iosStyles = StyleSheet.create({
```
Comment (colincclark) [inline packages/pull-quote/styles/index.android.js]: should these be called androidStyles?

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/e3df661c0beea0b7

**Pull request merged**